### PR TITLE
Prep for Identity preview release

### PR DESCRIPTION
**Pull request by christothes**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/identity/Azure.Identity.Broker/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.2.0-beta.1 (2024-04-17)
 
 ### Other Changes
+- The `IsProofOfPossessionRequired` property on `InteractiveBrowserCredentialBrokerOptions` and `SharedTokenCacheCredentialBrokerOptions` has been removed. This property has moved to the experimental `PopTokenRequestContext` struct.
```
Comment (xiangyan99) [inline sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: Is this a breaking change?

**Review by KarishmaGhiya — APPROVED** [review]

File: sdk/identity/Azure.Identity.Broker/CHANGELOG.md (hunk 1 above)
Comment (christothes) [reply sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: no, because it has never been GA'd

Comment (KarishmaGhiya) [reply sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: it's an overload method. so may be not?

Comment (christothes) [reply sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: I'll switch it to the breaking change section, just to be safe

**Pull request merged**